conda_build/exceptions.py

```diff
@@ -48,3 +48,11 @@ def error_body(self):
 
 class UnableToParseMissingSetuptoolsDependencies(CondaBuildException):
     pass
+
+
+class VerifyError(RuntimeError):
+    def __init__(self, error, script, *args):
+        self.error = error
```

Comment:
Hmmm - why is this not being tested?  Can we write a test case for a VerifyError and make sure that it is thrown and has the info you want?